src/translations/Bahasa_Indonesia.js

```diff
@@ -0,0 +1,93 @@
+﻿/**

```

Comment:
According to the linter's check, line endings have to be LF and not CRLF

You also have to [link the translation](https://github.com/Monogatari/Monogatari/blob/93b384f0228e90e0cc39915dc0d5d658559ffbd5/src/index.js#L42) with monogatari's core

Two possible actions you have to do : 
1. > mnafisalmukhdi1 wants to merge 1 commit into Monogatari:develop from **unknown repository**

    Seems like you didn't forked the project, or you deleted it. Since I can't have access to the repository that contains your changes, I can't correct line endings by myself. Try to restore it if possible, or fork + commit again the changes + create another PR.
2. I fork this project, add translations + register them and create a new pull request. It will just have my name as the submitter of the PR, but you will of course be considered as the main translator for that language, and credited on the contributors page.

Please tell me what option you'd like to choose. If you are interested in programming, I recommend the first one because mastering the concept of git is important. The community is here to guide you.

Reply:
1. Yes, I've delete my repository but I'll fork it again, commit the changes and do the pull request. Also contact you for asking help about the line ending suppose to be LF.